elasticsearch/src/test/java/org/apache/metamodel/elasticsearch/ElasticSearchDataContextTest.java

```diff
@@ -559,7 +560,16 @@ private static void indexBulkDocuments(String indexName, String indexType, int n
 
     }
 
-    private static void indexOneTweeterDocumentPerIndex(String indexType, int id) {
+    private static void indexTweeterDocument(String indexType, int id, Date date) {
+        try {
+            client.prepareIndex(indexName, indexType).setSource(buildTweeterJson(id, date))
+                    .setId("tweet_" + indexType + "_" + id).execute().actionGet();
+        } catch (Exception ex) {
+            System.out.println("Exception indexing documents!!!!!");
```

Comment:
I completely agree (except I wouldn't rethrow, but use throws instead), but it seemed to be the pattern for all the indexing methods here, so I just went with the flow :)